cookbook/import_export/main-concepts.rst

```diff
@@ -0,0 +1,62 @@
+Main Concepts
+=============
+
+Like your catalog, your data sources, channels and business rules are unique.
+
+This is why a common task is to work on connectors to import and export the PIM data as expected.
+
+Akeneo PIM comes with a set of configurable connectors based on re-usable classes and services.
+
+.. image:: ./akeneo_overview.svg
+
+Overview
+--------
+
+The Import/Export system is based on Akeneo BatchBundle.
+
+It helps to define "high level" jobs such as imports, exports and bulk actions.
+
+There are the main objects of the architecture.
+
+.. image:: ./batch-main-concepts.png
+
+.. note::
+
+    Akeneo BatchBundle is very inspired by Spring Batch http://spring.io/docs, it implements a very small part of the original work, and mainly provides reusable functions to process large volumes of records.
+
+Connector
+---------
+
+A Connector is packaged as a Symfony bundle.
+
+It contains classes, services and configurations to register new Jobs in Akeneo PIM.
+
+Job
+---
+
+The main batch domain object, it represents an explicit abstraction representing the configuration of a job specified by a developer.
```

Comment:
This sentence is very complicated. May be this would be easier to understand : "Being the main batch domain object, a job is an explicit abstraction representing its configuration specified by a developer".